tests/Queue/QueueWorkerTest.php

```diff
@@ -77,11 +77,11 @@ public function test_exception_is_reported_if_connection_throws_exception_on_job
         $this->exceptionHandler->shouldHaveReceived('report')->with($e);
     }
 
-    public function test_worker_sleeps_when_queue_is_empty()
```

Comment:
@dusterio - no, `runNextJob()` is indirectly used by `queue:listen` - which is why this test exists